feature/security/gnsi/authz/tests/authz_test.go

```diff
@@ -0,0 +1,752 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package authz_test performs functional tests for authz service
+package authz_test
```

Comment:
Did you test this PR in your local environment? In your local environment, do you have proxy in the middle?

Reply:
we have tcp proxy (transparent proxy) and for it to work, we simply include proxy address in the ips of the router certificate.  Also, (as TODO) our current mtls implementation in FP test needs to be changed to authticate based on spiffie id and in that case if router uses svid, the proxy should not impact the connection.